src/test/resources/expected_create_creditcard_payment.json

```diff
@@ -2,6 +2,7 @@
   "amount": 1.0,
   "description": "Description",
   "redirectUrl": "redirectUrl",
+  "webhookUrl": "webhookUrl",
```

Comment:
Could you add a test where the `webhookUrl` is an `Optional.empty()` and check if it works properly?